R/args.R

```diff
@@ -944,6 +947,10 @@ validate_pathfinder_args <- function(self) {
   if (!is.null(self$num_draws)) {
     self$num_draws <- as.integer(self$num_draws)
   }
+  checkmate::assert_integerish(self$num_psis_draws, lower = 1, null.ok = TRUE, len = 1)
```

Comment:
I just remembered that there's a `.var.name` argument to the `assert_*` functions that lets you change the name that's displayed to the user in the error message. We could use that here so that the user sees a message about `draws` instead of `num_psis_draws`